src/api/audits/methods.test.ts

```diff
@@ -155,6 +161,11 @@ describe('audit methods', () => {
         expect(puppeteer.launch).toHaveBeenCalledWith(
           expect.objectContaining({
             executablePath: 'some/path',
+            args: [
+              '--incognito',
+              '--no-sandbox',
+              '--remote-debugging-port=1000',
```

Comment:
should this not override even if they specified chrome port? no strong pref just curious